inst/qml/PenalizedMetaAnalysis.qml

```diff
@@ -0,0 +1,415 @@
+//
+// Copyright (C) 2013-2018 University of Amsterdam
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with this program.  If not, see
+// <http://www.gnu.org/licenses/>.
+//
+import QtQuick			2.8
+import QtQuick.Layouts	1.3
+import JASP.Controls	1.0
+import JASP.Widgets		1.0
+import JASP				1.0
+import "../qml/qml_components" as MA
+
+Form
+{
+	VariablesForm
+	{
+		preferredHeight: 400 * preferencesModel.uiScale
+
+		AvailableVariablesList
+		{
+			name: "allVariables"
+		}
+
+		AssignedVariablesList
+		{
+			name:				"effectSize"
+			title:				qsTr("Effect Size")
+			singleVariable:		true
+			suggestedColumns:	["scale"]
+		}
+
+		AssignedVariablesList
+		{
+			name:				"standardError"
+			title:				qsTr("Effect Size Standard Error")
+			singleVariable:		true
+			suggestedColumns:	["scale"]
+		}
+
+		DropDown
+		{
+			name:			"method"
+			id:				method
+			label:			qsTr("Method")
+			currentIndex:	2
+
+			values: [
+				{ label: qsTr("Horseshoe"),	value: "hs"		},
+				{ label: qsTr("Lasso"),		value: "lasso"	}
+			]
+		}
+
+/*		AssignedVariablesList
+		{
+			name:				"studyLabels"
+			title:				qsTr("Study Labels")
+			singleVariable:		true
+			suggestedColumns:	["ordinal", "nominal"]
+		}
+*/
```

Comment:
yeah, the current implementation by the author does not allow plot the penalized estimates so it would not be useful. It might be added in the future so I would leave it commented out for now.